qiskit/transpiler/passes/calibration/base_builder.py

```diff
@@ -68,7 +68,7 @@ def run(self, dag: DAGCircuit) -> DAGCircuit:
                 # calibration can be provided and no user-defined calibration is already provided
                 try:
                     schedule = self.get_calibration(node.op, qubits)
-                except CalibrationNotAvailable:
+                except CalibrationNotAvailableError:
                     # Fail in schedule generation. Just ignore.
                     continue
```

Comment:
Seems like this isn't really an error either.

Reply:
Reverted